db/event_manager_test.go

```diff
@@ -475,16 +481,21 @@ func TestWebhookOldDoc(t *testing.T) {
 	if !testLiveHTTP {
 		return
 	}
-	count, sum, _ := InitWebhookTest()
+
+	count, sum, _, ts := InitWebhookTest()
+	defer ts.Close()
+	url := ts.URL
+
 	ids := make([]string, 200)
 	for i := 0; i < 200; i++ {
 		ids[i] = fmt.Sprintf("%d", i)
 	}
 
 	time.Sleep(1 * time.Second)
 	eventForTest := func(i int) (Body, string, base.Set) {
+		idx := Abs(int64(i))
```

Comment:
I see that the original motivation to support eventForTest(-i) was to have different values for oldBody and body, and that this is fixing a bug in that original implementation, but this feels like an unnecessarily complicated way of doing that.  It would be more straightforward to just switch eventForTest to take two values (string and int), for id and value.